src/tests/suite/dfs_unit_test.c

```diff
@@ -993,38 +998,37 @@ dfs_test_compat(void **state)
 	if (arg->myrank != 0)
 		return;
 
-	uuid_generate(uuid1);
-	uuid_clear(uuid2);
-
 	print_message("creating DFS container with set uuid "DF_UUIDF" ...\n", DP_UUID(uuid1));
-	rc = dfs_cont_create(arg->pool.poh, uuid1, NULL, NULL, NULL);
+	rc = dfs_cont_create(arg->pool.poh, &uuid1, NULL, NULL, NULL);
```

Comment:
this test is about checking backward compatibility with the API changes we made, so i would like to not change anything here to make sure we didn't break API/ABI.